### PR TITLE
MINOR: Improve postProcessAndValidateIdempotenceConfigs method

### PR DESCRIPTION
**Pull request by RivenSun2**
Improve postProcessAndValidateIdempotenceConfigs method

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by RivenSun2** [comment]
Hi @showuon @guozhangwang 
could you help to review this PR?
Thanks.

**Comment by RivenSun2** [comment]
Hi @showuon 
could you help to review this PR?
Thanks a lot.

File: clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java [hunk 1]
```diff
@@ -490,11 +490,11 @@ private void postProcessAndValidateIdempotenceConfigs(final Map<String, Object>
                 if (userConfiguredIdempotence) {
                     throw new ConfigException("Must set " + RETRIES_CONFIG + " to non-zero when using the idempotent producer.");
                 }
-                log.info("Idempotence will be disabled because {} is set to 0.", RETRIES_CONFIG, retries);
+                log.info("Idempotence will be disabled because {} is set to 0.", RETRIES_CONFIG);
```
Comment (showuon) [inline clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java:493]: Nice catch!

**Review by showuon — APPROVED** [review]
LGTM! Thanks for the improvement!

**Comment by showuon** [comment]
@RivenSun2 , I don't know why the jenkins doesn't run the test. Could you rebase to the trunk branch to trigger the test again? Thanks.

**Comment by RivenSun2** [comment]
Hi @showuon 
Thanks for your review.
I just retriggered jenkins build.
Thanks.

**Comment by showuon** [comment]
Failed tests are unrelated:
```
    Build / JDK 17 and Scala 2.13 / kafka.api.ConsumerBounceTest.testSeekAndCommitWithBrokerFailures()
    Build / JDK 8 and Scala 2.12 / kafka.server.KRaftClusterTest.testSetLog4jConfigurations()
```

**Comment by showuon** [comment]
@RivenSun2 , thanks for the contributions! 
One little suggestion to you. I've noticed that each time, your PR will include all the unrelated commits. (ref: https://github.com/apache/kafka/pull/12069/commits)  It doesn't affect the result, though, I think it is better to learn to make the commit history clear. 

tip: you should always fork out another branch based on the latest trunk branch and then start the code change. FYI.

**Comment by RivenSun2** [comment]
@showuon Ah, thanks for your suggestion.
I'm here to learn how you or others create PRs.
Thank you very much.

**Pull request merged**